src/Bulb/Bulb.php

```diff
@@ -23,6 +23,10 @@ class Bulb
     const ADJUST_PROP_BRIGHTNESS = 'bright';
     const ADJUST_PROP_COLOR_TEMP = 'ct';
     const ADJUST_PROP_COLOR = 'color';
+    const POWER_MODE_CURRENT = 0;
```

Comment:
```suggestion
    const POWER_MODE_CURRENT = 0;
    const POWER_MODE_CT = 1;
    const POWER_MODE_RGB = 2;
    const POWER_MODE_HSV = 3;
    const POWER_MODE_COLOR_FLOW = 4;
    const POWER_MODE_NIGHT_LIGHT = 5;
```
WDYT about adding all supported modes and naming them according to yeelight doc?